packages/react-icons/src/utils/useIconState.tsx

```diff
@@ -6,7 +6,9 @@ const useRootStyles = makeStyles({
         display: 'inline',
         lineHeight: 0,
 
-        "@media screen and (-ms-high-contrast: black-on-white)": {fill: 'windowtext'}
+        "@media (forced-colors: active)": {
```

Comment:
I think we could be more specific and only target the `svg` element here. It's the actual element that has the color inherit issue

https://github.com/w3c/csswg-drafts/issues/6310
https://bugs.chromium.org/p/chromium/issues/detail?id=1242706